discovery/src/main/java/com/ibm/watson/discovery/v1/model/QueryOptions.java

```diff
@@ -36,7 +36,6 @@
   private Long passagesCharacters;
   private Boolean deduplicate;
   private String deduplicateField;
-  private String collectionIds;
```

Comment:
Just wanted to check - should we have `spelling_suggestions` here? not sure if this was included with the def you generated with

Reply:
im guessing not since we dont have autocomplete here either

Reply:
Nope, don't have that change here. I'll have it in for the final merge.